specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/applicationGateway.json

```diff
@@ -452,6 +452,70 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/applicationGateways/{applicationGatewayName}/backendHealthOnDemand": {
```

Comment:
POST action names should start with a verb, such as: getBackendHealth or listBackendHealth

Reply:
changed to getBackendHealthOnDEmand